pkg/sidecar/identity/member.go

```diff
@@ -73,6 +85,16 @@ func (m *Member) GetSeeds(ctx context.Context, coreClient v1.CoreV1Interface, ex
 
 	if len(otherPods) == 0 {
 		// We are the only one, assuming first bootstrap.
+
```

Comment:
How about we early return at the beginning if rackOrdinal and podOrdinal == 0? I think it would make this logic a little bit cleaner, and it would save us one listing - cheap one but still.

Reply:
I've implemented it that way initially but it breaks replacement - a node can't be its own seed if its replacing a node, so in a single DC (no external seeds) it'd cause Scylla to fail on bootstrap.

Reply:
I see, it makes sense. Lets leave it as is then.